avm/src/vm/run.rs

```diff
@@ -93,21 +94,29 @@ pub async fn run_file(fp: &str, delete_after_load: bool) {
   if delete_after_load {
     std::fs::remove_file(Path::new(fp)).unwrap();
   }
-  run(bytecode, 8000).await;
+  run(bytecode, HttpType::HTTP(HttpConfig { port: 8000, })).await;
 }
 
-pub async fn run_agz_b64(agz_b64: &str) {
+pub async fn run_agz_b64(agz_b64: &str, priv_key_b64: Option<&str>, cert_b64: Option<&str>) {
```

Comment:
nit we should leave a comment that this method is almost exclusively called by the alan daemon